public/css/style.css

```diff
@@ -135,3 +135,38 @@ section {
   border-radius: 20px;
   background-color: rgb(244, 246, 247);
 }
+.blog-edit-container {
+  max-width: 400px;
+  margin: 30px;
+  padding: 10px;
+  border: 1px solid rgba(0,0,0,.5);
+  border-radius: 20px;
+  background-color: rgb(244, 246, 247);
+}
+.blog-text-input {
+  margin: 15px;
+}
+#blog-body-input {
+  display: flex;
```

Comment:
This flex layout is completely unnecessary, it looks identical if it is changed to block (and all of the flex specific CSS is removed)